test_runner/src/test/kotlin/ftl/ios/XctestrunTest.kt

```diff
@@ -17,6 +19,8 @@ import java.nio.file.Paths
 class XctestrunTest {
 
     private val swiftXctestrun = "$fixturesPath/EarlGreyExampleSwiftTests_iphoneos13.4-arm64e.xctestrun"
+    private val multipleTargetsSwiftXctestrun = "$fixturesPath/axel/AllTests_iphoneos13.7-arm64e.xctestrun"
```

Comment:
@jan-gogo In order to test new functionality, I have to add some new fixtures, but we should wait for this PR first:
https://github.com/Flank/flank/pull/1148, right?

Reply:
> @jan-gogo In order to test new functionality, I have to add some new fixtures, but we should wait for this PR
> #1148, right?

Right